actionpack/test/assertions/response_assertions_test.rb

```diff
@@ -84,7 +95,8 @@ def test_error_message_shows_302_redirect_when_302_asserted_for_success
         @response.location = 'http://test.host/posts/redirect/1'
 
         error = assert_raises(Minitest::Assertion) { assert_response :success }
-        expected = "Expected response to be a <success>, but was a <302>" \
+        expected = "Expected response to be a <2XX: success>,"\
```

Comment:
It looks kind of weird that 'success' is lowercase, while the actual response codes are uppercase, but I don't want to make it seem like 'Success' is a real HTTP response name.